src/components/Publish/_utils.ts

```diff
@@ -535,22 +537,19 @@ export async function verifyRawServiceCredential(
 
         if (rootService.length > 1) {
           return {
-            verified: true,
-            complianceApiVersion,
+            ...verifiedAndComplianceApiVersion,
             idMatch: rootService?.includes(did || did.toLowerCase()),
```

Comment:
issue(blocking): the statement will not work for lowerCase check, as the logical or is not supported in this way. An alternative could be to move the `||` outside the function parameter:
`rootService.includes(did) || rootService.includes(did.toLowerCase())`